### ISSUE TITLE
HEBREW ACCENT TELISHA GEDOLA trope mark is missing

### ISSUE THREAD

**Issue by etayluz** [akivajgordon/tikkun.io#99]
In tikkun.io - Genesis 37:7 - the first word is displayed as such
<img width="89" alt="Screen Shot 2021-11-24 at 3 12 54 PM" src="https://user-images.githubusercontent.com/1825834/143307177-36049aca-add6-46ec-9b96-42a52a0eaace.png">

However on Sefaria it appears as such
https://www.sefaria.org/Genesis.37.9?lang=he&aliyot=0:
![image](https://user-images.githubusercontent.com/1825834/143307237-3be52dcd-34cb-4f87-a2e1-3d99570f7b09.png)

Any native Hebrew speaker will know that Sefaria's version is correct. The accent should be applied to the NUN and the secondary pashta accent is missing there.

I imagine there are numerous typos like this.
What is the source of the text?

I do NOT trust it

**Comment by akivajgordon** [comment]
@etayluz – thanks for creating this issue!

Yes, it's been frustrating for me as well – I've seen a ton of these errors all over within the source text, which is ironically acquired from Sefaria, e.g.: https://github.com/Sefaria/Sefaria-Export/blob/master/json/Tanakh/Torah/Genesis/Hebrew/Tanach%20with%20Ta'amei%20Hamikra.json

From that source, the verse is given as:

```
            "וְ֠הִנֵּה אֲנַ֜חְנוּ מְאַלְּמִ֤ים אֲלֻמִּים֙ בְּת֣וֹךְ הַשָּׂדֶ֔ה וְהִנֵּ֛ה קָ֥מָה אֲלֻמָּתִ֖י וְגַם־נִצָּ֑בָה וְהִנֵּ֤ה תְסֻבֶּ֙ינָה֙ אֲלֻמֹּ֣תֵיכֶ֔ם וַתִּֽשְׁתַּחֲוֶ֖יןָ לַאֲלֻמָּתִֽי׃",
```

So I'd need to get a new source of the text. If you know where I can get access to a machine-parsable (i.e. JSON) format with the corrections, I'd love if you can point me to it.

Thanks!

**Comment by etayluz** [comment]
What I've noticed is that the text as it appears on Sefaria's website in very much correct and complete. However - their Sefaria-Export dump seems to be a completely different version that is actually based on [tanach.io](https://www.tanach.us/) - which is the source of all the errors.

I'll put together a nice json file and attach it later. It will just be a copy and paste of the content from Sefaria's website.

**Comment by akivajgordon** [comment]
Fixed with #102
